AppConnectSwift/CreateAccountViewController.swift

```diff
@@ -46,7 +47,18 @@ class CreateAccountViewController: UIViewController {
     
     func showAlert(title: String, message: String) {
         let alert = UIAlertController(title: title, message: message, preferredStyle: UIAlertControllerStyle.Alert)
-        alert.addAction(UIAlertAction(title: "OK", style: UIAlertActionStyle.Default, handler: nil))
+        if title.containsString("Account Creation Success"){
```

Comment:
Can you pass the handler into the `showAlert` method, to avoid special logic in this method? So `showAlert` can just take the title, message, and a handler to call once the user clicks 'OK'.